### PR TITLE
gh-103088: Fix scripts for Windows

### PR DESCRIPTION
**Pull request by Zabolekar**
* Issue: gh-103088

### CODE REVIEW

File: Misc/NEWS.d/next/Windows/2023-04-11-09-22-22.gh-issue-103088.6AJEuR.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Fixes venvs not working in bash on Windows across different disks
+
+PCbuild\build.bat now can use an existing Python 3.11 installation
```
Comment (zooba) [inline Misc/NEWS.d/next/Windows/2023-04-11-09-22-22.gh-issue-103088.6AJEuR.rst]:
```suggestion
```

This doesn't need to go into NEWS (and if it did, it'd go into the "Build" category)

Comment (Zabolekar) [reply Misc/NEWS.d/next/Windows/2023-04-11-09-22-22.gh-issue-103088.6AJEuR.rst]: Ok, removed the line.

**Comment by rdbisme** [comment]
Can we cut another release (I'm interested for Python 3.10) that includes this?

**Comment by pfmoore** [comment]
No, sorry, Python 3.10 is now in security fix only mode - see [the release schedule](https://peps.python.org/pep-0619/#release-schedule).

**Comment by rdbisme** [comment]
@zooba any chance we might backport this to 3.11? We're 1 version behind at work, and forced to use windows, and while using Git bash or Msys and creating venvs, doing `python -m pip install <something>` with a venv activated installs the packages in the main python `site-packages` instead.

I'd appreciate for it to work as expected :)

**Comment by zooba** [comment]
Backporting to 3.11 is probably okay, but we need a new PR without the additional change to the batch file.

If you (or someone else) creates one, target 3.11 and put `[3.11]` at the start of the PR name and `(GH-103325)` at the end.

**Pull request merged**